### ISSUE TITLE
Null value causes Property to fail with object has no public Property or Member named [...]

### ISSUE THREAD

**Issue by doug-episource** [codingseb/ExpressionEvaluator#67]
Having a Dynamic Object with a null value for a Property results in the following error:

LogParsing.ExpressionEvaluatorSyntaxErrorException: '[System.Dynamic.ExpandoObject] object has no public Property or Member named "NullValue".'

```
            dynamic MyDynamic = new System.Dynamic.ExpandoObject();
            MyDynamic.NullValue = null;

            ExpressionEvaluator evaluator = new ExpressionEvaluator();
            evaluator.Variables = new Dictionary<string, object>()
            {
                  { "md", MyDynamic },
            };
            string expression = "md.NullValue ?? \"A\"";
            var exp = evaluator.Evaluate(expression);
```
Verified on the latest version of the code. Any ideas?

**Comment by codingseb** [comment]
Hello @doug-episource thanks for this issue.

Yes it was a bug in the way of evaluating properties on an ExpandoObject, I just published version 1.4.16.0 that correct it.

**Comment by doug-episource** [comment]
Great news! Thank you so much for the quick response and fix for the the issue!
